pkg/operator/ceph/object/objectstore.go

```diff
@@ -358,3 +361,75 @@ func GetObjectBucketProvisioner(c *clusterd.Context, namespace string) string {
 	}
 	return provName
 }
+
+func checkDashboardUser(context *Context) bool {
+	args := []string{"dashboard", "get-rgw-api-access-key"}
+	cephCmd := ceph.NewCephCommand(context.Context, context.ClusterName, args)
+	out, _ := cephCmd.Run()
+	if string(out) == "" {
+		return false
+	} else {
+		return true
+	}
+}
+
+func enableRGWDashboard(context *Context) error {
+	if checkDashboardUser(context) {
+		logger.Debug("dashboard already enabled for rgw")
+		return nil
+	}
+	dispname := fmt.Sprintf("%s", DashboardUser)
```

Comment:
Why? Isn't `DashboardUser` enough?

Reply:
Oh I see, then make it a var instead of a constant?

Reply:
Done